src/components/GlobalSearch.jsx

```diff
@@ -10,40 +11,62 @@ function GlobalSearch({
   onClick,
   searchValue,
   handleClear,
-  img
+  img,
+  searchCount,
+  setSearchCount
 }) {
+  const handleErrorClose = () => {
+    setSearchCount(false);
```

Comment:
if it a single line change, keep it inline where you invoke this.
No need for a separate function call.